### ISSUE TITLE
Fine-tune

### ISSUE THREAD

**Issue by mehrdad78** [Dragonisss/RAM#4]
Hi
how can i fine-tune one of pretrained model with my own pair images?
Thank you

**Comment by DragonisCV** [comment]
Hi, Thank you for your interest in RAM. Could you please give me some details about your idea?

**Comment by mehrdad78** [comment]
Thank you for your quick response ,
How should i set the path to train and test set ?

**Comment by DragonisCV** [comment]
1. First, you need to collect your datasets above and place them under the `./datasets/` Directory.Symbolic links is a recommended approach, allowing you to place the datasets anywhere you prefer!
2. You need to rewrite a Dataset class in "./ram/data". May be you can directly use [ PairedImageDataset](https://github.com/DragonisCV/RAM/blob/main/ram/data/paired_image_dataset.py) class to build your own dataloader.
3. Modify your dataset options in "./options/base/**.yaml". You can reference [our datasets setting](https://github.com/DragonisCV/RAM/blob/main/options/base/dataset/ram_train_dataset.yaml).

Our repo is borrows from [BasicSR](https://github.com/XPixelGroup/BasicSR). you can also check it to learn more about our codebase.
